backend/code/archipelag/templates/registration/event.html

```diff
@@ -16,34 +16,36 @@ <h1 class="title">Stwórz Wydarzenie</h1>
                 <form id="event_form" class="main-login main-center col-md-4 col-md-offset-4"
                     style="font-size:17px">{% csrf_token %}
                     <div class="md-form">
-                        <label for="form1" class="">Tytuł</label>
+                        <label for="title" class="">Tytuł</label>
                         <input type="text" id="title" class="form-control" >
```

Comment:
`id` attribute is for a DOM element. Could you replace `id` attribute as `name`?

Reply:
but label need id

Reply:
Ok, Forgive me comment :P